src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -2253,14 +2253,16 @@
 
 helps['storage share-rm delete'] = """
 type: command
-short-summary: Delete the specified Azure file share.
+short-summary: Delete the specified Azure file share or share snapshot.
 examples:
   - name: Delete an Azure file share 'myfileshare' under the storage account 'mystorageaccount' (account name) in resource group 'MyResourceGroup'.
     text: az storage share-rm delete -g MyResourceGroup --storage-account mystorageaccount --name myfileshare
   - name: Delete an Azure file share 'myfileshare' under the storage account 'mystorageaccount' (account id).
     text: az storage share-rm delete --storage-account mystorageaccount --name myfileshare
   - name: Delete an Azure file share by resource id.
     text: az storage share-rm delete --ids file-share-id
+  - name: Delete an Azure file share snapshot.
```

Comment:
We are hesitated to add delete command because it may have breaking change.

Reply:
Discussed with yishi offline, this is for deletion of specific snapshot.  Breaking change is for delete share with all snapshots.